src/backend/SyncBackendConnection.cpp

```diff
@@ -1124,11 +1193,13 @@ bool SyncBackendConnection::update_one_entity_in_model_(
         // Get item from model from the index already calculated
         ListItem* item = model->at(index);
 
-        // Get alive status of entity.
+        // Get alive status and metatraffic attribute of entity.
```

Comment:
```suggestion
        // Get alive status and metatraffic attribute of entity. Take it from Backend and not from item because item has not been uploaded yet.
```

Reply:
:+1: